spec/helpers/create-test-lib.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+var URI;
+
+URI = require("urijs");
+
+if (!jasmine.createTestLib) {
```

Comment:
No reason apparently. I thought this was executed multiple times (for example once per test file). This is apparently not the case.